client/src/components/Burger/Burger.css

```diff
@@ -0,0 +1,41 @@
+.Burger > div {
+    height: 8px;
+    width: 50px;
+    background-color: var(--black);
+    margin: 8px 0;
+    transition: .1s ease-in-out;
+    -webkit-transition: .1s ease-in-out;
+    -moz-transition: .1s ease-in-out;
+    -ms-transition: .1s ease-in-out;
+    -o-transition: .1s ease-in-out;
+}
```

Comment:
Can you move this to the `.Burger__line` class? That will be more explicit

Reply:
Done.